packages/engine.test/src/testView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Engine } from "@c11/engine.react";
+import { generateImage } from 'jsdom-screenshot';
+
+jest.useFakeTimers();
```

Comment:
The same as above, make sure that this only runs in the scope of a specific test and ensure you restore the real timers.
https://testing-library.com/docs/using-fake-timers